AIM/AIM/Domain/Entities/StationInfo.h

```diff
@@ -0,0 +1,24 @@
+//
+//  StationInfo.h
+//  AIM
+//
+//  Created by Oskar Szydlowski on 04.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface StationInfo : NSObject
+
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic, strong) NSString *description;
+@property (nonatomic, strong) NSString *time;
```

Comment:
what time is that one? time of a stationInfo? since you have displayTime i'd  consider another name for this property